conf/uppmax.config

```diff
@@ -1,11 +1,17 @@
-// Profile config names for nf-core/configs
+// UPPMAX Config Profile
 params {
-  config_profile_description = 'Swedish UPPMAX cluster profile provided by nf-core/configs.'
+  config_profile_description = 'UPPMAX (Rackham) cluster profile provided by nf-core/configs.'
```

Comment:
This is not only Rackham since it works on Irma and Bianca too

Reply:
Lower down, Irma and Snowy are accounted for.

Reply:
I'll add a code comment noting that